sktime/forecasting/pytorchforecasting.py

```diff
@@ -378,3 +522,520 @@ def get_test_params(cls, parameter_set="default"):
+        still needed, but no real fitting will be performed.
+    random_log_path: bool (default=False)
+        use random root directory for logging. This parameter is for CI test in
+        Github action, not designed for end users.
+
+    Examples
+    --------
+    >>> # import packages
+    >>> from sktime.forecasting.base import ForecastingHorizon
+    >>> from sktime.forecasting.pytorchforecasting import PytorchForecastingNHiTS
+    >>> from sktime.utils._testing.hierarchical import _make_hierarchical
+    >>> # generate random data
+    >>> data = _make_hierarchical(
+    ...     hierarchy_levels=(5, 200), max_timepoints=50, min_timepoints=50, n_columns=3
+    ... )
+    >>> # define forecast horizon
+    >>> max_prediction_length = 5
+    >>> fh = ForecastingHorizon(range(1, max_prediction_length + 1), is_relative=True)
+    >>> # split X, y data for train and test
+    >>> l1 = data.index.get_level_values(1).map(lambda x: int(x[3:]))
+    >>> X_train = data.loc[l1 < 190, ["c0", "c1"]]
+    >>> y_train = data.loc[l1 < 190, "c2"].to_frame()
+    >>> X_test = data.loc[l1 >= 180, ["c0", "c1"]]
+    >>> y_test = data.loc[l1 >= 180, "c2"].to_frame()
+    >>> len_levels = len(y_test.index.names)
+    >>> y_test = y_test.groupby(level=list(range(len_levels - 1))).apply(
+    ...     lambda x: x.droplevel(list(range(len_levels - 1))).iloc[:-max_prediction_length]
+    ... )
+    >>> # define the model
+    >>> model = PytorchForecastingNHiTS(
+    ...     trainer_params={
+    ...         "max_epochs": 5,  # for quick test
+    ...         "limit_train_batches": 10,  # for quick test
+    ...     },
+    ... )
+    >>> # fit and predict
+    >>> model.fit(y=y_train, X=X_train, fh=fh) # doctest skip
+    PytorchForecastingNHiTS(trainer_params={'limit_train_batches': 10,
+                                            'max_epochs': 5})
+    >>> y_pred = model.predict(fh, X=X_test, y=y_test)
+    >>> print(y_test)
+                                c2
+    h0   h1     time
+    h0_0 h1_180 2000-01-01  8.184178
+                2000-01-02  5.444128
+                2000-01-03  5.992600
+                2000-01-04  5.223143
+                2000-01-05  6.191883
+    ...                          ...
+    h0_4 h1_199 2000-02-10  7.498591
+                2000-02-11  5.910466
+                2000-02-12  7.409602
+                2000-02-13  4.670040
+                2000-02-14  5.454403
+
+    [4500 rows x 1 columns]
+    >>> print(y_pred)
+                                c2
+    h0   h1     time
+    h0_0 h1_180 2000-02-15  5.764410
+                2000-02-16  5.826406
+                2000-02-17  5.925301
+                2000-02-18  5.792100
+                2000-02-19  5.760923
+    ...                          ...
+    h0_4 h1_199 2000-02-15  5.376267
+                2000-02-16  5.227071
+                2000-02-17  5.070744
+                2000-02-18  5.249713
+                2000-02-19  5.047630
+
+    [500 rows x 1 columns]
+    >>>
```

Comment:
why are these three `>>>` here. If not required by dostring, please remove them. This applies to all of the examples.

Reply:
removed